docs/guide/migrationV2.md

```diff
@@ -8,3 +8,4 @@ A number of changes were made to the configuration options passed to the plugin
 
  * `xScaleID` option default has been changed, now set to `undefined`. If the option is missing, the plugin will try to use the first scale of the chart, configured as `'x'` axis. If more than one scale has been defined in the chart as `'x'` axis, the option is mandatory to select the right scale.
  * `yScaleID` option default has been changed, now set to `undefined`. If the option is missing, the plugin will try to use the first scale of the chart, configured as `'y'` axis. If more than one scale has been defined in the chart as `'y'` axis, the option is mandatory to select the right scale.
+ * The scale dimension, not chart area anymore, is used to set `xMin`, `xMax`, `yMin` and `yMax` options when missing. This enables the stacked scale management.
```

Comment:
Should add a link to the chart.js docs for "stacked scale", because it might be confused with normal stacking.
Also could be worth mentioning that this only affects charts with stacked scales.

Reply:
have a look if sounds better now